src/swsssdk/port_util.py

```diff
@@ -61,6 +61,9 @@ def get_interface_oid_map(db):
     """
         Get the Interface names from Counters DB
     """
+    if_name_map = {}
+    if_id_map = {}
```

Comment:
@liorghub please elaborate how does it solve the issue. Seems like some logic is missing here

Reply:
@nazariig 
Before the fix, when there are no ports and no lags in the system, if_name_map was None and we had an exception when trying to access its content (if_name_map.items()).
Declaring it solved the issue, causing the function to return an empty dictionary as it should.

Same goes for if_id_map.
We declare an empty dictionary, it stays empty (since we put data there only if there is data in if_name_map, line 81) and being returned.

Reply:
We discussed over the phone.